pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2165,6 +2166,84 @@ var _ = Describe("migratableDomXML", func() {
 	})
 })
 
+var _ = Describe("Manager helper functions", func() {
+
+	Context("getVMIEphemeralDisksTotalSize", func() {
+
+		var tmpDir string
+		var zeroQuantity resource.Quantity
+
+		BeforeEach(func() {
+			var err error
+			tmpDir, err = ioutil.TempDir("", "tempdir")
```

Comment:
Here it's better to use `os.TempDir()`, the `ioutil` package has been deprecated as of Go 1.16.

Reply:
Thanks! Didn't know that it was deprecated :)

p.s. it's `os.MkdirTemp()`